### PR TITLE
including SJCampos

### PR DESCRIPTION
**Pull request by julioarruda**

### CODE REVIEW

File: 2023/dotnetvale/data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "CommunityInfo": {
+    "Name": ".Net Vale",
+    "Country": "Brazil",
+    "Url": "https://www.sympla.com.br/evento/global-azure-bootcamp/1894910?_gl=1*yr5avl*_ga*OTk2NDE2MDUxLjE2NzYxNDc4NzQ.*_ga_KXH10SQTZF*MTY3NzQzMzUwMi4xMS4xLjE2Nzc0MzM3MTYuMC4wLjA",
+    "SessionizeApiId": "10800",
+    "Description_SessionizeApiId": "10800",
+    "SessionizeEventName": "global-azure",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_template.png"
```
Comment (noopman) [inline 2023/dotnetvale/data.json:19]: You need to include your community logo! If you do not have one, some communities have made a simple image with their community name on it!

File: 2023/dotnetvale/data.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
```
Comment (noopman) [inline 2023/dotnetvale/data.json:5]: We need the GitHub handles of anyone who is allowed to modify your community folder "dotnetvale", including yours!

File: 2023/dotnetvale/data.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": ".Net Vale",
+    "Country": "Brazil",
+    "Url": "https://www.sympla.com.br/evento/global-azure-bootcamp/1894910?_gl=1*yr5avl*_ga*OTk2NDE2MDUxLjE2NzYxNDc4NzQ.*_ga_KXH10SQTZF*MTY3NzQzMzUwMi4xMS4xLjE2Nzc0MzM3MTYuMC4wLjA",
+    "SessionizeApiId": "10800",
+    "Description_SessionizeApiId": "10800",
```
Comment (noopman) [inline 2023/dotnetvale/data.json:15]: Is this Sessionize ID right? @davidjrh how do I verify this?

File: 2023/dotnetvale/data.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "SessionizeEventName": "global-azure",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_template.png"
+    },
+    "Locations": [
+      {
+        "Name": "{Name of the location}",
+        "Address": "{Optional address of the location}",
```
Comment (noopman) [inline 2023/dotnetvale/data.json:24]: You don't have a named location for your event yet?

**Review by noopman — CHANGES_REQUESTED** [review]
There is a community logo file, but that is just the template file. The logo for the community is missing. Please add a logo!

I am also missing a .MD file in this folder to publicly describe the community. Please look in other folders next to yours to see what other communities have done here!

File: 2023/dotnetvale/data.json (hunk 2 above)
Comment (julioarruda) [reply 2023/dotnetvale/data.json:5]: Ok, i really forgot this

File: 2023/dotnetvale/data.json (hunk 3 above)
Comment (julioarruda) [reply 2023/dotnetvale/data.json:15]: I put a wrong value.. sorry.. I never used a Sessionize before.. I think that is correct now.

File: 2023/dotnetvale/data.json (hunk 2 above)
Comment (noopman) [reply 2023/dotnetvale/data.json:5]: Something is wrong. I cannot see any changes!

File: 2023/dotnetvale/data.json (hunk 3 above)
Comment (noopman) [reply 2023/dotnetvale/data.json:15]: Something is wrong. I cannot see any changes!

**Review by noopman — CHANGES_REQUESTED** [review]
Something is wrong. I cannot see any changes!

File: 2023/dotnetvale/data.json (hunk 4 above)
Comment (julioarruda) [reply 2023/dotnetvale/data.json:24]: The local is not exactly defined yet, but probably this will be the place.

**Review by noopman — CHANGES_REQUESTED** [review]
Great! Now I see some of your changes, and they look good! Great logo!

Let's ditch the unused file _template.png also, and then we're done!

Nice work!

File: 2023/dotnetvale/data.json (hunk 3 above)
Comment (julioarruda) [reply 2023/dotnetvale/data.json]: Can you see now? I answered this before the commit, in my github.dev 🤦‍♂️

**Comment by julioarruda** [comment]
Ok, I removed the _template.png

Thanks

**Review by noopman — APPROVED** [review]
Perfection! <3 Thank you for working so diligently on this! Welcome to Global Azure 2023!

**Pull request merged**